test/test_sparse.py

```diff
@@ -1503,6 +1503,14 @@ def test_log1p(self):
             device=self.device)
         self._test_log1p_tensor(input, torch.zeros([5, 6, 0]))
 
+    def test_mv(self):
+        i = self.index_tensor([[0, 1, 1], [2, 0, 2]])
+        v = self.value_tensor([3, 4, 5])
+        x = self.sparse_tensor(i, v, torch.Size([2, 3]))
+        y = torch.ones(3, device=self.device)
```

Comment:
You might want to use different values for each element, not all ones, just to be safe. In fact you could think about using larger randomly generated tensors, with multiple different test cases. Check out `test_mm()` in this file.

Reply:
done